src/libs/Url.js

```diff
@@ -19,7 +19,15 @@ function addTrailingForwardSlash(url) {
  */
 function getURLObject(href) {
     const urlRegex = new RegExp(URL_WEBSITE_REGEX, 'gi');
-    const match = urlRegex.exec(href);
+    let match;
+    try {
+        if (!href.startsWith('mailto:')) {
```

Comment:
I tried it but I tend to add this improvement for the following reasons. 
1. The slow regex execution takes some time to throw the error, which means the UI will be blocked during the time.
2. As we render the long email as anchor tag, so every time we open a chat includes a long email will result in a brief moment of UI block. Given that I think it's better to skip executing the regex for email here.

Reply:
I think early return for the email case is enough to avoid executing slow regex matching in this method. We avoid parsing long link causes error to anchor tag in method ExpensiMark.replace, which means we will only possibly have long email.

Reply:
> Can we not add a similar condition (which we use while parsing long link) here as well?

I think we can't because we don't do domain(TLD) checking when parsing email in ExpensiMark